benchmarks/eko_bench.py

```diff
@@ -95,10 +95,21 @@ def benchmark_sv(self, pto):
         high = base_theory.copy()
         high["PTO"] = pto
         high["fact_to_ren_scale_ratio"] = np.sqrt(2.0)
-        high["ModSV"] = "A"
+        high["ModSV"] = "exponentiated"
         # needed for apfel
         high["XIR"] = np.sqrt(0.5)
         high["EScaleVar"] = 0
+
+        # here we need to adjust also the
+        # apfel initial nf, which we can't
+        # be accessed in other ways
+        # Be careful: this is and hack not a proper solution
```

Comment:
1) "a hack and" :see_no_evil: 
2) so we're only testing the bottom threshold?

Reply:
Great that you finally succeeded! So bad APFEL..

I would rename the file to `scale_variations.py`. It is useful and I'd keep it, but `eko_bench.py` is a bit too generic...

Reply:
but I believe @giacomomagni is using this in a generic way - which happens to work also for SV ... correct me if I'm wrong @giacomomagni

Reply:
Yes, I'm using it in a generic way, but actually I'm not satisfied by this trick. Because as you can see the problem is just shift on T24 and happens to be less visible. But this is **not** the solution, I believe.

Reply:
Actually, I tried to run it, having reinstalled dependencies as they are in `poetry.lock`, but I get the following error:
```
ValueError: 48 columns passed, passed data had 49 columns
```
Clearly coming from db...

By chance, are you using your `banana` branch or something like this?

Reply:
Nope, did you pull? Because I fixed a typo `ModSv`, which was causing this issue,  in one of the lasts commit.

Reply:
I pulled `eko`, but I was relying on the released `banana` version.

If you wish, this time let me just finish as soon as possible the current batch, and we definitely release a new version. However, in general, we should try to stick as much as possible to released versions.

I've the first one to use `develop` branches together, but we're getting more and more stable (and we are also more and more developers with more and more machines), so better to use as much as possible releases, otherwise it's a mess to reproduce the environment (while with releases `poetry` is really helping us a lot).

Reply:
Okay, got it! In this case it was not possible due to the additional label `ModSv`. But I agree with you.

Reply:
yes, I'd say so - let's write the issue and forget